src/client/Microsoft.Identity.Client/TelemetryCore/TelemetryClient/TelemetryDatapoints.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.Cache;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.TelemetryCore.TelemetryClient
+{
+    /// <summary>
+    /// Stores details to log to the <see cref="ITelemetryClient"/>.
+    /// </summary>
+    public class TelemetryDatapoints
+    {
+        /// <summary>
+        /// Type of cache used. This data is captured from MSAL or Microsoft.Identity.Web to log to telemetry.
+        /// </summary>
+        public CacheTypeUsed CacheTypeUsed { get; set; } = CacheTypeUsed.None;
```

Comment:
What does `Used` mean? Cache read? Read or write? Or if L2 is set up at all? The life time of this class is per request, but both L1 and L2 can be used.

Reply:
Yea, that makes sense, I can seperate the values so both levels can be set.

Reply:
Actually, i added a new entry to the enum to indicate when both caches are used.